scripts/evaluate.py

```diff
@@ -1213,43 +1289,51 @@ def get_args():
     )
 
     args = parser.parse_args()
-    if args.log == None:
-        args.log = "logs/" + args.model + "." + args.type + ".autotvm.log"
-    if args.rpc_tracker_port != None:
-        args.rpc_tracker_port = int(args.rpc_tracker_port)
-    args.tuning_options = {
-        "log_filename": args.log,
-        "early_stopping": None,
-        "measure_option": autotvm.measure_option(
-            builder=autotvm.LocalBuilder(build_func=ndk.create_shared, timeout=15, n_parallel=2),
-            runner=autotvm.RPCRunner(
-                args.rpc_key,
-                host=args.rpc_tracker_host,
-                port=args.rpc_tracker_port,
-                number=50,
-                timeout=15,
-                #min_repeat_ms=150,
-                #cooldown_interval=150
+    
+    if not args.build_and_export:
+        if args.log == None:
+            args.log = "logs/" + args.model + "." + args.type + ".autotvm.log"
+        if args.rpc_tracker_port != None:
+            args.rpc_tracker_port = int(args.rpc_tracker_port)
+        args.tuning_options = {
+            "log_filename": args.log,
+            "early_stopping": None,
+            "measure_option": autotvm.measure_option(
+                builder=autotvm.LocalBuilder(build_func=ndk.create_shared, timeout=15, n_parallel=2),
+                runner=autotvm.RPCRunner(
+                    args.rpc_key,
+                    host=args.rpc_tracker_host,
+                    port=args.rpc_tracker_port,
+                    number=50,
+                    timeout=15,
+                    #min_repeat_ms=150,
+                    #cooldown_interval=150
+                ),
             ),
-        ),
-    }
+        }
+    else:
+        args.tuning_options = {             
+        }
```

Comment:
why did you wrap all parameters?
the problem was only with `tuning_options`, you only need to wrap it.